### PR TITLE
Eq 1945 social alpha

### PR DESCRIPTION
**Pull request by davecharles**
### What is the context of this PR?
Author a first cut survey schema for Labour Force Survey.  This PR adds a new schema for LFS to `data/en`, there are no code changes.

Only the first path of the LFM design has been implemented (Socio Demographic), the remainder (Labour Market) will be implemented in a subsequent iteration.  Issues and omissions have been documented on the related [Trello card](https://trello.com/c/eBUSUODf).

### How to review 
- Check schema is valid using the schema validator i.e:
   `pipenv run ./scripts/test_schemas.sh data/en/labour_force.json`
- Run the survey [with this url](https://preprod-surveys-launch.eq.ons.digital/quick-launch?url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-1945-social-alpha/data/en/labour_force.json).

### CODE REVIEW

**Comment by dwyeradam** [comment]
A `yarn format` should fix your linting issue.

File: data/en/labour_force.json [hunk 1]
```diff
@@ -0,0 +1,3201 @@
...
+                                    "mandatory": false,
+                                    "type": "MonthYearDate"
+                                },
+                                {
+                                    "id": "arrive-in-uk-alt-answer",
+                                    "alias": "arrived_alt",
+                                    "mandatory": false,
+                                    "type": "Radio",
+                                    "options": [{
+                                            "label": "Don&apos;t know",
```
Comment (MIslam92) [inline data/en/labour_force.json]: Should we be allowing the respondent to give a date then answer "I don't know"/"Prefer not to say " at this point? I know these are essentially two different answers and the mutually exclusive answers might overcome this.

File: data/en/labour_force.json [hunk 2]
```diff
@@ -0,0 +1,3201 @@
...
+                    "hide_in_navigation": true,
+                    "routing_rules": [{
+                        "repeat": {
+                            "type": "answer_count_minus_one",
+                            "answer_id": "first-name"
+                        }
+                    }],
+                    "blocks": [{
+                        "type": "Question",
+                        "id": "household-relationships",
```
Comment (MIslam92) [inline data/en/labour_force.json:142]: Should this not route to asking the relationship question when the household members are equal to two as well? It takes this into consideration in the JSON but doesn't match what the docs are saying in the card?

**Review by MIslam92 — CHANGES_REQUESTED** [review]
Just some observations in the comments

File: data/en/labour_force.json (hunk 1 above)
Comment (davecharles) [reply data/en/labour_force.json]: Agreed with @charlotte-hirst to remove the DK/PNTS radios for now.

File: data/en/labour_force.json (hunk 2 above)
Comment (davecharles) [reply data/en/labour_force.json:142]: The routing design is based on a separate "first page" to get info on first two household occupants and their relationship.  We couldn't do this with the existing household-composition mechanics from census so @charlotte-hirst agreed we could duck this in the alpha.

**Review by MebinAbraham — COMMENTED** [review]
Shouldn't this section be selected in the navigation?
image

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
If I click `Continue` on the proxy check question. I get the following.
image

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Ignore this. :point_down:

**Comment by shahi645** [comment]
@MebinAbraham The above is because there is piping in the answer value

**Comment by davecharles** [comment]
I can get around this by rephrasing the option labels but it seems to be an issue with functionality (i.e. this should work). @shahi645 sounds like this is a known issue?

**Review by MebinAbraham — APPROVED** [review]
Need to raise a card for the issue above, but apart from that, looks good.

**Comment by nealedj** [comment]
Note that https://github.com/ONSdigital/eq-survey-runner/pull/1533 changes the way that values are referenced when piping. If it gets merged first then this PR will need updating -> or vice versa.

**Comment by davecharles** [comment]
@MebinAbraham raised #1538

**Review by MIslam92 — APPROVED** [review]

**Comment by shahi645** [comment]
@davecharles Rebase and Squash and we can merge this one?

**Pull request merged**